packages/grafana-ui/src/components/Logs/logParser.ts

```diff
@@ -22,6 +22,11 @@ export const getAllFields = memoizeOne(
     const fields = parseMessage(row.entry);
     const derivedFields = getDerivedFields(row, getFieldLinks);
     const fieldsMap = [...derivedFields, ...fields].reduce((acc, field) => {
+      if (!field.links && row.labels[field.key] !== undefined) {
```

Comment:
Do you think it would be viable to use JS [`in` operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in) here?

It performs quite the same on small objects, but as soon as the object gets larger, the performance is better:
<img width="579" alt="image" src="https://user-images.githubusercontent.com/8092184/190578721-897e96c8-68c6-4b9d-a7c7-bdbcbed3a136.png">